core-services/document-grounding/src/main/resources/spec/grounding.yaml

```diff
@@ -350,24 +604,25 @@ paths:
   "/vector/collections/{collectionId}/documents":
     get:
       tags:
-        - vector
+        - Vector
       summary: Get documents
       description: Gets a list of documents of a collection.
       operationId: vector.v1.vector_endpoints.get_all_documents
-      x-sap-cloud-sdk-operation-name: getAllDocuments
       parameters:
```

Comment:
Why do our generated service method names look _fubar_? Because colleagues have [removed our spec extension.](https://github.tools.sap/AI/rage-proxy-inference/pull/325/)


```diff
  final var documents = CLIENT_VECTOR
-   .getAllDocuments(RESOURCE_GROUP, collectionId);
+   .vectorV1VectorEndpointsGetAllDocuments(RESOURCE_GROUP, collectionId);
```